omnibus-optimism-mainnet.toml

```diff
@@ -117,3 +117,8 @@ args = [
         { marketId = "<%= extras.synth_eth_market_id %>", weightD18 = 1, maxDebtShareValueD18 = "<%= parseEther('1') %>" },
     ],
 ]
+
+[clone.trusted_multicall_forwarder]
+source = "trusted-multicall-forwarder"
```

Comment:
Please set the version and preset. Or we risk this being redeployed, which we don't want.